protostar/commands/init/project_creator/new_project_creator.py

```diff
@@ -120,7 +120,8 @@ def _new_project_config(
 
     @staticmethod
     def _validate_project_name(name: str):
-        pattern = r"^[a-zA-Z0-9][a-zA-Z0-9_]*$"
+        # https://github.com/software-mansion/scarb/blob/main/scarb/src/core/package/name.rs#L26
+        pattern = r"^[a-zA-Z_][0-9a-zA-Z_]*$"
```

Comment:
So we can have package names starting with multiple underscores? 🤔

Reply:
We actually can, it just cannot be equal to `_`.
Fixed in [update the regex and docs](https://github.com/software-mansion/protostar/pull/1849/commits/62128d330cb55dc846b5bd5353155256987f8b6f).
You can look up the Scarb PR https://github.com/software-mansion/scarb/pull/302